### DISCUSSION TITLE
Network ports

### DISCUSSION THREAD

**Discussion by petanque** [KRTirtho/spotube#1857] [Q&A] [answered]
I'm using Spotube behind a network bridge and I'm unsure which ports are being used, is it possible that that info can be shared?

**Comment by KRTirtho** [comment]
The ports are random to prevent attacks on local networks. But the service name is discoverable via zeroconf. The talk name/type is `_spotube._tcp`

**Reply by KRTirtho** [reply, marked as answer]
https://github.com/python-zeroconf/python-zeroconf can be used to find the service. Some scripting will be needed

**Reply by petanque** [reply]
It makes sense because I kept finding new ports.
I have to see if I have the skills to make a script or if I have to come up with another solution.
Thank you for your quick reply.
